server/implementation/src/main/java/io/smallrye/graphql/bootstrap/Bootstrap.java

```diff
@@ -339,9 +340,20 @@ private void createGraphQLEnumType(EnumType enumType) {
         GraphQLEnumType.Builder enumBuilder = GraphQLEnumType.newEnum()
                 .name(enumType.getName())
                 .description(enumType.getDescription());
+        // Directives
+        if (enumType.hasDirectiveInstances()) {
+            enumBuilder = enumBuilder.withDirectives(createGraphQLDirectives(enumType.getDirectiveInstances()));
+        }
         // Values
         for (EnumValue value : enumType.getValues()) {
-            enumBuilder = enumBuilder.value(value.getValue(), value.getValue(), value.getDescription());
+            GraphQLEnumValueDefinition.Builder definitionBuilder = GraphQLEnumValueDefinition.newEnumValueDefinition()
+                    .name(value.getValue())
```

Comment:
It looks strange that the `name` and `value` both are set to `getValue()`, but it was like this before, so it's not getting worse. I don't understand what graphql-java is doing here.

Reply:
I though exactly the same: looks suspicous but I won't change it in this PR.

Reply:
As far as I can tell `value` should hold the "Java runtime value" for the respective GraphQL enum value, i.e. it is used to map the GraphQL enum value to the Java enum value when executing a query. See the [comment in EnumValuesProvider.java](https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/schema/idl/EnumValuesProvider.java#L6). It would probably be safe to not set it here since we're not executing any queries. At least the SchemaPrinter uses only the name, not the value.

Reply:
👍thanks for the explanation. maybe a comment would be helpful?

Reply:
The above explanation is my educated guess after diving into the code for about 10 minutes. Someone set the `value` for a reason, maybe it needs to be set to a value to omit an NullPointerException somewhere. I'd have to check if we could remove setting `value`, but that would be a different PR. I don't want to break things in this PR that are outside of the scope of supporting directives on enums.

Reply:
yeah, sure. I just thought that a comment could help the next person reading the line to understand what's happening.

Reply:
I looked into the issue and set `value`to null and had a couple of tests failing, so it looks like we need it like it was. I also added some comments.